install/install.sh

```diff
@@ -41,13 +41,15 @@ usage() {
     * Specify a database management system. Valid options include: postgres, sqlite (if not specified, the default is to use sqlite).
       If the database management system is specified to use postgres, the following extra variables must also be set: QPC_DBMS_HOST, QPC_DBMS_PASSWORD
```

Comment:
I feel like we should probably reword some of this documentation because everything starts with "If using postgres," . . . and the default is now to use postgres. We need more of a specification between having a preconfigured postgres & creating an internal one.  This is mostly me rambling, but see the comments below.